### DISCUSSION TITLE
Looking for sony 5.1ch Home Theatre IR Codes

### DISCUSSION THREAD

**Discussion by afkcodes** [probonopd/irdb#181] [Code requests] [answered]
Hello this is particularly not an issue but i was looking for IR Codes for my old home theatre. 12 years tbh. 
Model SA-ID5000
Remote Model : RM-ANU088
Tried looking on to website got pretty much confused, any help would be appreciable. Also can anyine help me how to look into this huge DB and find the models. I actually dont know what sony calls it home theatre or something other.

**Comment by probonopd** [comment]
I would not be surprised if this is what it is using:

![image](https://user-images.githubusercontent.com/2480569/121804570-422e3180-cc47-11eb-9490-cb57070f845c.png)

Did you try this with IrScrutinizer yet?

**Comment by probonopd** [comment, marked as answer]
Also check out https://github.com/probonopd/irdb/issues/47.
